tests/console/firewalld.pm

```diff
@@ -18,6 +18,10 @@ use testapi;
 use utils qw(systemctl zypper_call);
 use version_utils qw(is_sle is_leap is_tumbleweed);
 
+sub has_iptables {
```

Comment:
Fabian suggested calling it `uses_iptables`

Reply:
done